webpack.config.js

```diff
@@ -59,7 +59,7 @@ const config = (module.exports = {
   output: {
     path: BUILD_PATH + "/app/dist",
     // NOTE: the filename on disk won't include "?[chunkhash]" but the URL in index.html generated by HtmlWebpackPlugin will:
-    filename: "[name].bundle.js?[hash]",
```

Comment:
# Replace deprecated `[hash]` in the config
```
[frontend] ℹ ｢wds｣: Project is running at http://0.0.0.0:8080/
[frontend] ℹ ｢wds｣: webpack output is served from http://localhost:8080/app/dist/
[frontend] ℹ ｢wds｣: Content not from webpack is served from frontend
```

Loading plugins succeeded
```
[backend] 2021-05-17 16:31:27,127 INFO metabase.core :: Metabase Initialization COMPLETE
```

Errors:
```
[frontend] ✖ ｢wdm｣: Entrypoint app-main = 999.hot.bundle.js?c0e03a66f81dc341c630 66.hot.bundle.js?2592bdeea40df0a3f32b 358.hot.bundle.js?6c5b3dd6e16a099eeddb 968.hot.bundle.js?65f1efd13fcfb47c152d app-main.hot.bundle.js?3292e6c8d8b2b4e1ea0f 5 auxiliary assets
[frontend] Entrypoint app-public = 999.hot.bundle.js?c0e03a66f81dc341c630 66.hot.bundle.js?2592bdeea40df0a3f32b 968.hot.bundle.js?65f1efd13fcfb47c152d 417.hot.bundle.js?8f5b08420e667863af0e app-public.hot.bundle.js?6419673902b0e68bb825 5 auxiliary assets
[frontend] Entrypoint app-embed = 999.hot.bundle.js?c0e03a66f81dc341c630 66.hot.bundle.js?2592bdeea40df0a3f32b 968.hot.bundle.js?65f1efd13fcfb47c152d 417.hot.bundle.js?8f5b08420e667863af0e app-embed.hot.bundle.js?c8a820f3f83504b744c2 5 auxiliary assets
[frontend] Entrypoint styles = 999.hot.bundle.js?c0e03a66f81dc341c630 styles.hot.bundle.js?0f3376f9117758e07c81 2 auxiliary assets
```

Warnings
```
[frontend] WARNING in DefinePlugin
[frontend] Conflicting values for 'process.env.NODE_ENV'
```

```
[frontend] WARNING in configuration
[frontend] The 'mode' option has not been set, webpack will fallback to 'production' for this value.
[frontend] Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
[frontend] You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/configuration/mode/
```

```
[frontend] 1 WARNING in child compilations (Use 'stats.children: true' resp. '--stats-children' for more details)
```

```
[frontend] 1 warning has detailed information that is not shown.
[frontend] Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.
```

Error:
```
[frontend] ERROR in ../../../node_modules/babel-plugin-transform-react-display-name/lib/index.js 95:12-27
[frontend] Module not found: Error: Can't resolve 'path' in '/Users/offbynone/code/work/metabase/node_modules/babel-plugin-transform-react-display-name/lib'
```

```
[frontend] BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
[frontend] This is no longer the case. Verify if you need this module and configure a polyfill for it.
[frontend] 
[frontend] If you want to include a polyfill, you need to:
[frontend]      - add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
[frontend]      - install 'path-browserify'
[frontend] If you don't want to include a polyfill, you can use an empty module like this:
[frontend]      resolve.fallback: { "path": false }
[frontend]  @ ../../../node_modules/babel-preset-react/lib/index.js 17:44-96
[frontend]  @ ./internal/components/ScratchApp.jsx 40:24-53
[frontend]  @ ./internal/components/ sync (\w+)App.jsx$ ./ScratchApp.jsx
[frontend]  @ ./internal/routes.js 46:10-80
[frontend]  @ ./routes.jsx 465:24-67
[frontend]  @ ./app-main.js 7:14-40
```